lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -241,18 +245,25 @@ def move_predicate(corrector, actual, matcher, block_child)
           corrector.insert_after(actual, ".#{predicate}" + args + block)
         end
 
+        TO_PREDICATE_METHOD_MAP = {
+          a_kind_of:         'is_a?',
+          an_instance_of:    'instance_of?',
+          be_a:              'is_a?',
+          be_a_kind_of:      'is_a?',
+          be_an:             'is_a?',
+          be_an_instance_of: 'instance_of?',
+          be_instance_of:    'instance_of?',
+          be_kind_of:        'is_a?',
+          include:           'include?',
+          respond_to:        'respond_to?'
+        }.freeze
+        private_constant :TO_PREDICATE_METHOD_MAP
+
         def to_predicate_method(matcher)
-          case matcher = matcher.to_s
-          when 'be_a', 'be_an', 'be_a_kind_of', 'a_kind_of', 'be_kind_of'
```

Comment:
Subjectively, this reads better than multiple hash items

Reply:
I changed it to a hash lookup because one of the Metric cops said the method became too long. But I agree with you, and I will change it back when/if I get back to this PR.